server/pension/models.py

```diff
@@ -25,6 +25,87 @@ class InstrumentType(object):
         (NON_NEGOTIABLE_SECURITY, _('Non-Negotiable Security')),
     )
 
+class Industry(object):
+    UNDEFINED = 0
+    PHARMA = 1
+    HI_TECH = 2
+
+    choices = (
+        (UNDEFINED, _('Undefined')),
+        (PHARMA, _('Pharma')),
+        (HI_TECH, _('Hi-Tech')),
+    )
+
+class Currency(object):
```

Comment:
@ranmoshe Please add `docs`, what it is, what should be here, which model it's "serve"?